models/issues/issue.go

```diff
@@ -596,6 +596,8 @@ func IsUserParticipantsOfIssue(ctx context.Context, user *user_model.User, issue
 type DependencyInfo struct {
 	Issue                 `xorm:"extends"`
 	repo_model.Repository `xorm:"extends"`
+	IssueID               int64 `xorm:"NOT NULL"`
```

Comment:
document: IssueID is the id of the issue the that is used for the selection of dependent or blocked issues